app/Http/helpers.php

```diff
@@ -221,3 +233,179 @@ function getIndianCurrencyFormat($number)
         return $thecash;
     }
 }
+
+function bifurcateTax($taxName, $taxValue, $currency, $state, $price)
```

Comment:
write proper tests for significant methods like `bifurcateTax`

Reply:
Fixed